ting_subsearch_secondary.module

```diff
@@ -99,46 +99,51 @@ function ting_subsearch_secondary_execute_secondary_search($keys, $results) {
   ]);
 
   if ($secondary_result->numTotalObjects > variable_get('ting_subsearch_secondary_result_min', 100)) {
-    $items = [];
+    $ratio = $secondary_result->numTotalObjects / $results->getNumTotalObjects();
+    if ($ratio > variable_get('ting_subsearch_secondary_ps_factor', 2)) {
+
+      $items = [];
+      $number_of_records_to_show = variable_get('ting_subsearch_secondary_number_of_results', 4);
+      foreach ($secondary_result->collections as &$collection) {
+        if ($number_of_records_to_show == 0) {
+          continue;
+        }
+        else {
+          $number_of_records_to_show--;
```

Comment:
These two dashes on purpose?

Reply:
Two dashes decrements the value by one.

Reply:
I see that original implementation uses this variable as a counter to how many times to render an object, yet I wonder why it wasn't implemented with a `brake` statement instead when `$number_of_records_to_show` reaches zero..